app/com/arpnetworking/kairos/client/models/SamplingUnit.java

```diff
@@ -53,4 +57,18 @@
      * Sampling is count represents years.
      */
     YEARS;
+
+    /**
+     * Encode the enumeration value as lower case.
+     *
+     * NOTE: KairosDb accepts either upper or lower case representation. This
+     * model is otherwise a pass-through but converts all enumeration values
+     * to lower case.
+     *
+     * @return json encoded value
+     */
+    @JsonValue
+    public String toJson() {
+        return name().toLowerCase(Locale.getDefault());
+    }
```

Comment:
nit: If Kairos isn't picky about casing, why do we need this method?

Reply:
You are correct, we don't. However, based on the documentation (which admittedly is not great) most of the examples were lower case. So given the decision of LC+UC=>UC or LW+UC=>LC I elected the latter which seemed more consistent with the documentation.